packages/asset-swapper/src/utils/market_operation_utils/index.ts

```diff
@@ -92,10 +88,25 @@ export class MarketOperationUtils {
                 this._liquidityProviderRegistry,
             ),
         );
+        const rfqtPromise =
+            opts !== undefined && _opts.rfqt !== undefined && _opts.rfqt.quoteRequestor !== undefined
+                ? _opts.rfqt.quoteRequestor.requestRfqtIndicativeQuotesAsync(
+                      nativeOrders[0].makerAssetData,
+                      nativeOrders[0].takerAssetData,
+                      takerAmount,
+                      MarketOperation.Sell,
+                      _opts.rfqt,
+                  )
+                : Promise.resolve([] as RfqtIndicativeQuoteResponse[]);
```

Comment:
Can you do `Promise.resolve<RfqtIndicativeQuoteResponse[]>([])` instead?  Feels a bit cleaner than the typecasting done here

Reply:
My TypeScript ignorance precedes me.  Thank you for the idiom.  I've fixed this here: https://github.com/0xProject/0x-monorepo/pull/2555/commits/d6d4d29257c4cb0810c280f8edc33939f0e6092e#diff-bdb1465a97651026c09ef607e5de9e6fR105